README.md

```diff
@@ -1,107 +1,134 @@
 # Managed Service Account
 
-## What is Managed Service Account?
+## Overview
 
-"Managed Service Account" is an OCM addon developed over [addon-framework](https://github.com/open-cluster-management-io/addon-framework)
-for synchronizing [ServiceAccount](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/)
-to the managed clusters and collecting the tokens from these local service
-accounts as secret resources back to the hub cluster. This addon will be 
-helpful when you're:
+The Managed Service Account addon is an Open Cluster Management (OCM) addon built on the [addon-framework](https://github.com/open-cluster-management-io/addon-framework). It synchronizes [ServiceAccount](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/) resources to managed clusters and collects their authentication tokens back to the hub cluster as secret resources.
 
-- Ensuring service account resources to the managed clusters w/o a kubeconfig
-  to the managed cluster.
-- Accessing the kube api of the managed clusters from the hub cluster which
-  will require legit authentication tokens.
-- Homogenizing the client identity to the same service account when requesting
-  the managed clusters' api.
-  
-The addon basically consists of two components following the typical 
-architecture of an OCM [addon](https://open-cluster-management.io/concepts/addon/):
+## Use Cases
 
-- "Addon-Manager": Automatically installs the addon agent into the managed 
-  cluster and related required resources.
-  
-- "Addon-Agent": Watching the "ManagedServiceAccount" API and projecting the 
-  service account token periodically as secret resources to the hub cluster.
-  And refreshes the tokens as well according to the rotation policy.
+This addon is beneficial when you need to:
 
-## Install
+- Deploy service account resources to managed clusters without requiring a kubeconfig to each managed cluster
+- Access the Kubernetes API of managed clusters from the hub cluster using valid authentication tokens  
+- Standardize client identity by using the same service account across managed cluster API requests
 
-### Prerequisite
+## Architecture
```

Comment:
The managed service account addon supports 2 installation ways:
- default, manager - agent
- addontemplate, only agent

Reply:
Thanks. Done as suggested.